installers/msi-language/Uninstall/CustomAction.cs

```diff
@@ -33,7 +57,22 @@ public static ActionResult Uninstall(Session session)
                 return ActionResult.Failure;
             }
 
-            return Remove.EnvironmentEntries(session, installDir);
+            result = Remove.EnvironmentEntries(session, installDir);
+            if (result.Equals(ActionResult.Failure))
+            {
+                session.Log("Could not remove shortcuts directory");
+                ActiveState.RollbarHelper.Report("Could not remove shortcuts directory");
```

Comment:
The remove above is not related to shortcuts.

Also nit: since we are creating the shortcuts in the preset logic it might also make sense to remove them there

Reply:
I actually decided not to move it into the `Preset.Uninstall()` method, because the `shortCutDir` is something that we want to remove for every language probably...  We'll see. :)